src/adapters/adyoulike.js

```diff
@@ -0,0 +1,177 @@
+import Adapter from 'src/adapters/adapter';
+import bidfactory from 'src/bidfactory';
+import bidmanager from 'src/bidmanager';
+import * as utils from 'src/utils';
+import { format } from 'src/url';
+import { ajax } from 'src/ajax';
+import { STATUS } from 'src/constants';
+
+var AdyoulikeAdapter = function AdyoulikeAdapter() {
+  const _VERSION = "0.1";
+
+  const baseAdapter = Adapter.createNew('adyoulike');
+  const bidRequests = {};
```

Comment:
Keeping state out here will be buggy. What would happen if `callBids` gets called twice before your ajax request returns? For example, suppose a page makes more than one cal to `requestBids`.

You can make it safe if you use `newResponseHandler(requests)` as your ajax callback (where `newResponseHandler(requests)` returns the `handleResponse` function, with the requests stored in the closure scope)